packages/botonic-plugin-flow-builder/src/index.ts

```diff
@@ -68,6 +68,16 @@ export default class BotonicPluginFlowBuilder implements Plugin {
     return content
   }
 
+  async getContentByCode(code: string): Promise<NodeComponent> {
+    const flow = await this.flow
+    const content = flow.nodes.find(node => {
+      if ('code' in node) return node.code === code
+      return undefined
+    })
```

Comment:
early return normally suggest to do this in the other order: https://medium.com/swlh/return-early-pattern-3d18a41bba8
```
if(exception) return 
// happy path
```
```suggestion
const content = flow.nodes.find(node => {
      if (!('code' in node)) return undefined
      return node.code === code
    })
```

Reply:
I have applied the change in the following PR (tracking)

`const content = flow.nodes.find(node =>
      'code' in node ? node.code === code : false
    )`